force-app/main/default/classes/GooglePlacesTest.cls

```diff
@@ -122,80 +131,51 @@ private with sharing class GooglePlacesTest {
         List<Account> schools = GooglePlacesTestData.createSchoolAccount(1);
         List<Account> churches = GooglePlacesTestData.createDuplicateChurchAccount(1);
         GooglePlacesTestData.createExistingCSP(1, schools, churches);
+        GoogleMockCallout callout = new GoogleMockCallout('validKey');
 
         Test.startTest();
-        Test.setMock(HttpCalloutMock.class, new GoogleMockCalloutValidKey());
+        Test.setMock(HttpCalloutMock.class, callout);
         List<GooglePlaces.DataTableWrapper> wraps = GooglePlaces.searchPlace(schools[0].Id, 56.1232, 81.123, 20, schools[0].Type);
         Test.stopTest();
         System.assert(wraps.size() == 0, 'If existing CSP, no new CSP is created.');
     }
 
-    public class GoogleMockCalloutValidKey implements HttpCalloutMock {
+    public class GoogleMockCallout implements HttpCalloutMock {
```

Comment:
@Julian88Tex I've re-factored all the different Callout classes into one Class that can set different response bodies.